tests/articles/test_comments.py

```diff
@@ -0,0 +1,123 @@
+from rest_framework import status
+from ..test_base import BaseTest
+from .test_category import TestCategory
+from rest_framework.test import APIClient
+
+
+class TestComment(BaseTest, TestCategory):
+    def setUp(self):
+        self.comment_data = {
+            "comment": {
+                "body": "His name was my name too."
+            }
+        }
+        self.second_comment_data = {
+            "comment": {
+                "body": "His name was my name too. This !"
+            }
+        }
+        self.registration_data = {
+            "user": {
+                "username": "user",
+                "email": "userstest@gmail.com",
+                "password": "Users@12345"
+            }
+        }
+        self.login_data = {
+            "user": {
+                "email": "userstest@gmail.com",
+                "password": "Users@12345"
+            }
+        }
+        self.article = {
+            "article": {
+                "title": "believer",
+                "description": "This test was created on womens day of 2019.",
+                "body": "I like to move it move it, I like to move it",
+            }
+        }
+        self.new_article = {
+            "article": {
+                "title": "believer",
+                "description": "This test was created on womens day of 2019.",
```

Comment:
@nyabongoedgar, This is a repetition of code. You can re-use much more from the Base_test. You are writing the same instances which are in the class that you inherit.